app/models/course.rb

```diff
@@ -5,6 +5,36 @@ class Course < ApplicationRecord
 
   validates :code, presence: true, uniqueness: { scope: :provider_id }
   validates :name, presence: true
+  validates :start_date, presence: true
+  validates :level, presence: true
+  validates :age_range, presence: true
+  validates :duration_in_years, presence: true
+  validates :qualification, presence: true
+
+  enum level: {
+    primary: 0,
```

Comment:
should we put these enums into the config the same way we do with the others to be consistent? Or are we only doing that for situations where they need to be reused with DTTP logic?

Reply:
I think the common approach up until now was because of DTTP. We change it later if need be.